medusa/cache.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, filename):
 
         :param filename:
         """
+        super(MutexLock, self).__init__(filename)
```

Comment:
File "/home/osmc/Medusa/medusa/cache.py", line 20, in __init__
    super(MutexLock, self).__init__(filename)
  File "/home/osmc/Medusa/lib/dogpile/cache/backends/file.py", line 300, in __init__
    raise NotImplementedError()
NotImplementedError